extract/test/test_bbox_merge.py

```diff
@@ -0,0 +1,26 @@
+from extract.bbox import BBox
+from extract.bbox_merge import bbox_merge
+from extract.coordinate import Coordinate
+
+
+def test_bbox_merge():
+    bboxes = [
+        BBox(Coordinate(466.681,788.160),Coordinate(472.219,801.755)),
```

Comment:
This test case is both too complex and too simple.

Too complex because having nine million points doesn't really tell us anything. If it can handle three points, it can handle nine million. Having >3 points just makes it harder to see whats happening. So, simplify that.

But this doesn't test the case where you've got a BBox that has no coordinates. Write another test for that.

All of the lower_right.y coordinates are the same here. That's probably realistic, in that for in most cases the bboxes you're looking at will be on the same line. But it's possible to have off-by-one errors where it's always returning the biggest-but-one coordinate; or it's always returning the first item in the list. You can't detect those if all the values are the same - you can't tell if it's actually returning the max, or if it just picked a value at random and that happened to be the right answer for the wrong reason.

So add another test case where the y values differ, and make sure it's returning the correct y value.